package.json

```diff
@@ -15,23 +15,41 @@
     "url": "https://github.com/microsoft/BotFramework-WebChat/issues"
   },
   "homepage": "https://github.com/microsoft/BotFramework-WebChat#readme",
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged --no-stash --config lint-staged.config.js"
-    }
-  },
   "keywords": [],
   "scripts": {
     "bootstrap": "lerna bootstrap --ci",
+    "browser": "node ./packages/test/harness/src/host/dev/index http://localhost:5001/__tests__/html/",
+    "browser:watch": "node-dev --no-notify --respawn ./packages/test/harness/src/host/dev/index http://localhost:5001/__tests__/html/",
     "build": "lerna run --ignore playground --stream build",
-    "eslint": "lerna run --parallel --stream eslint -- ",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "docker": "concurrently \"npm run docker:up\" \"npm run docker:watch\"",
+    "docker:ci": "npm run docker:up -- --detach",
+    "docker:down": "docker-compose -f docker-compose-wsl2.yml down --rmi all",
+    "docker:up": "docker-compose -f docker-compose-wsl2.yml down && docker-compose -f docker-compose-wsl2.yml up --build --scale chrome=4",
+    "docker:watch": "nodemon --exec \"docker-compose -f docker-compose-wsl2.yml build --quiet -- jest-server webchat2 && docker-compose -f docker-compose-wsl2.yml kill webchat2 && docker-compose -f docker-compose-wsl2.yml rm --force --stop jest-server webchat2 && docker-compose -f docker-compose-wsl2.yml up --detach -- jest-server webchat2 && echo\" --ext html,js,map --on-change-only --watch __tests__/html --watch packages/test/harness/dist --watch packages/test/page-object/dist --watch serve-test.json --watch packages/test/harness/src/host/jest-server",
+    "eslint": "npm run precommit",
+    "lint-staged": "lint-staged",
+    "precommit": "lerna run --parallel --stream precommit -- ",
+    "precommit:eslint:api": "cd packages && cd api && npm run precommit:eslint",
```

Comment:
Can we refactor all of these out into a scripts file? at the very least for the `*:command:package` scripts. This is too chaotic.

Reply:
(Discussed offline - gotta keep all these due to `lint-config` settings requirements. We are sad lol